README.md

```diff
@@ -3,16 +3,18 @@
 > NOTE: If you are looking for the new Azure DevOps CLI, see [vsts-cli](https://github.com/microsoft/vsts-cli)
 
 [![NPM version](https://badge.fury.io/js/tfx-cli.svg)](http://badge.fury.io/js/tfx-cli)
+#### Internal Deploy / Pull Request validation
+[![build passing](https://shfeldma.visualstudio.com/_apis/public/build/definitions/f4b6db46-e446-49f0-a424-0bfb52c0925d/2/badge)](https://shfeldma.visualstudio.com/_apis/public/build/definitions/f4b6db46-e446-49f0-a424-0bfb52c0925d/2/badge)
 
 Command utility for interacting with Microsoft Team Foundation Server and Azure DevOps Services (formerly VSTS). It is cross platform and supported on Windows, OS X, and Linux.
 
 ## Setup
 
-First, download and install [Node.js](http://nodejs.org) 4.0.x or later and NPM (included with the installer)
+First, download and install [Node.js](http://nodejs.org) 7.0.x or later and NPM (included with the installer or from sources)
```

Comment:
Node 7 isn't a release supported by Node. Minimum would be an even number like 8